### DISCUSSION TITLE
Water Management (and some more)

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#530] [Ideas]
Below several ideas for possible changes and improvements to the system, some inter-related.

**(1) Range of reservoirs to be limited by land features**, like rivers or mountains. Reservoirs essentially create a system of underground pipes to make fountains and baths work; overground it is believable (cut-and-cover), underground (tunnels) rather not.
To counteract the changes, (2), (3) and (5) below would help.

**(2) Water supply buildings** (updated):
- a Small Reservoir (2x2) with a range of maybe 6 tiles out in each direction instead of 10; possible upgrade to Small Cistern
- Pumping Station: 2x2 building next to water, connected to aqueducts or directly to reservoirs. Would be required to allow aqueducts to fill in the first place (there should always be a level gradient to have the water flow). No pipe coverage on its own, but requires workers. Maybe with two visual variants: one on the coastline (like a wharf) and one on solid ground (because sometimes rivers don't have a lot of straight sections)
- more Fountain sizes, like combining the effects of medium/large statues together with water coverage; maybe a fountain Square, with through roads like a granary

Reservoir (3x3) upgrades (doable from the building screen to not clutter the build menu):
- pumping facilities (needs x workers); not needed if reservoir is directly adjacent to a pumping station, but required for (5)
- upgrade to Cistern (covered reservoir): providing better desirability and requiring workers. Because it requires workers, I don't think the upgrade should be automatic based on area desirability
- a mention of the water quality/flow level enabling specific upgrades to counter (4)

For small reservoirs, due to the limited space, pumping/filtration/purification upgrades would not fit.

**(3) Water access next to river/sea**, so that baths and fountains work without a reservoir (like clay pits). Could work well together with Pumping Stations.

**(4) Water quality / aqueduct length**
Depending on length to source, there could be some degradation in water quality (causing plague) or maybe throughput (diminishing range of reservoir). This could also be dependent on the province/region, where the water could be more contaminated or more sediment-prone. To avoid the issue/s, reservoir upgrades are proposed under (2).

**(5) Reservoirs at higher elevations**
To supply water to higher elevations in a more believable way, there could be 2-level aqueducts. These can be built on top of existing aqueducts and should require something extra to build, like a foreman/architect and some resources (clay? or maybe a new "stone" resource, which could be used in the future for sculptures).
During construction, there could be some scaffolding to visually indicate the stage of completion.
The second-level aqueduct could also be built on the mountain, to allow the level jump. Once it reaches the next elevation, it can be continued with a regular aqueduct.
Additionally, a 2-level reservoir (staffed and with pumping facilities) would be needed.
Hand-sketch below:
![Screenshot_20211117-224731715_1](https://user-images.githubusercontent.com/93785920/142283703-be118fb6-1705-4eb6-811c-d25aeec3b48f.jpg)
These aqueducts could have a monthly maintenance fee, like the temple/mausoleum levies.

**(6) Enhancement to road system**
Combining the engineering involved in aqueducts and because we already have over-water bridges, it should not be impossible to connect valleys with mountains and mountains with each other, so I propose:
- a new type of (paved) road that can be built on top of an aqueduct that is adjacent to a 2-level aqueduct.
- the road can go directly on land (mountain level) if it hugs the aqueduct.
- with a ramp, the road could go from aqueduct-level to valley-level.

Addition: **(7) Water supply and Health Advisor**
The water supply/quality (4) and a corresponding population coverage % could be mentioned in a new section, for example:
- no water access : 5%
- poor (including well access) : 10%
- average : 35%
- good: 50%

Using some of the same qualifiers the advisors use elsewhere.
The population coverage is based on the number of occupants in houses supplied with that quality of water, where:
- the water quality of a fountain/bathhouse is given by the water quality of the closest reservoir (that's where the pipes come from)
- the water quality for a house is given by the best water quality of a fountain within range
- for houses at Large Casa and above (requiring Baths access), the water supply for the Bath whose worker last visited the house should also be taken into account; maybe by averaging the quality of the fountain and bath supply

Also, a similar population coverage % would be useful also for Baths, Barbers and Clinics, which currently don't have any indicators.

**Comment by thomasherena** [comment]
This is a very well thought-out quality-of-life suggestion, definitely gets my vote tbh

**Reply by PaxBotanica** [reply]
Im definetly for the water supply buildings update. it may be that the ability to build water service on higher ground interferes with deliberate map making choices using the current mechanics. perhaps that would need to be mitigated by adding a syphon supply building with its own range that must be adjacent to rocks to supply water to a 2nd syphon building. Map makers may have the ability to disable this building to retain their map design functionality.

Separately I would love to see an on ground sewer system that uses a similar mechanic to aqueducts and must connect to a water source. Latrines (if introduced) would have to connect to both the water supply and sewer networks.
